agents-api/tests/test_entries_queries.py

```diff
@@ -0,0 +1,53 @@
+"""Tests for entry queries."""
+
+from agents_api.autogen.openapi_model import CreateEntryRequest, CreateSessionRequest
+from agents_api.clients.pg import create_db_pool
+from agents_api.queries.entries.count_entries import count_entries
+from agents_api.queries.entries.create_entries import create_entries
+from agents_api.queries.sessions.create_session import create_session
+from ward import test
+
+from tests.fixtures import pg_dsn, test_agent, test_developer_id
+
+
+@test("query: count entries by developer")
+async def _(dsn=pg_dsn, developer_id=test_developer_id, agent=test_agent):
```

Comment:
let's add a test that tries to create 500 entries on the free tier and fails with 403